operate/common/src/main/java/io/camunda/operate/property/BackupProperties.java

```diff
@@ -11,12 +11,23 @@ public class BackupProperties {
 
   private String repositoryName;
 
+  private long snapshotTimeoutMillis = 0;
```

Comment:
Would it make more sense to express this in seconds ? I don't think someone will ever use sub-second timeout for backups. It removes 3 unnecessary trailing zeroes